conda/recipes/cuxfilter/meta.yaml

```
@@ -25,11 +25,11 @@ requirements:
     - cuda-version ={{ cuda_version }}
   run:
     - bokeh >=2.4.2,<=2.5
-    - cudf ={{ minor_version }}
+    - cudf ={{ version }}
```

Why was this changed? I think the previous form was correct. We don't want to pin on the full version because patches aren't necessarily released for every RAPIDS package.
```suggestion
    - cudf ={{ minor_version }}
```

---

reverted the change